kolibri/plugins/device/assets/src/views/AvailableChannelsPage/index.vue

```diff
@@ -171,7 +173,28 @@
       ]),
       allChannels() {
         // Need to de-duplicate channels in case user enters same token twice, etc.
```

Comment:
If the intent is to return something like `[...sortedInstalledChannels, ...otherChannels]`, maybe what you could do is compute the `otherChannels` array using something like `lodash/differenceBy` or `Array.prototype.filter` to remove the installed channels from `this.availableChannels` to produce `otherChannels`

Reply:
Yup, agreed. I don't know if I can directly use `lodash/differenceBy` since the elements between `sortedInstalledChannels` and `otherChannels` aren't the exact same, but I've cleaned it up to make use of `Array.prototype.filter`.